packages/api/src/controllers/notification.ts

```diff
@@ -0,0 +1,107 @@
+import { sendgridEmail } from "./helpers";
+import { frontendUrl, unsubscribeUrl } from "./user";
+import { db } from "../store";
+import { sendgridEmailPaymentFailed } from "./helpers";
+import { WithID } from "../store/types";
+import { Request } from "express";
+import { User } from "../schema/types";
+
+export const getUsageNotifications = async (
+  usagePercentages,
+  user: WithID<User>
+) => {
+  let notifications = [];
+
+  if (
+    usagePercentages.TotalUsageMins > 90 ||
+    usagePercentages.DeliveryUsageMins > 90 ||
+    usagePercentages.StorageUsageMins > 90
+  ) {
+    notifications.push({
+      type: "notification90",
+      title: "Usage Warning",
+      message: `Your usage is over 90% of your limit.`,
+    });
+  } else if (
+    usagePercentages.TotalUsageMins > 75 ||
+    usagePercentages.DeliveryUsageMins > 75 ||
+    usagePercentages.StorageUsageMins > 75
+  ) {
+    await db.user.update(user.id, {
```

Comment:
shouldnt you push a notification here? 

also, when are the `notifications.usage.notification75/90` checked?

Reply:
Oh right, it was mixed up, all db.set and all checks are done in the function `notifyUser`

```
export const notifyUser = async (notifications, user, req) => {
```